tests/common_parameters.py

```diff
@@ -19,16 +19,16 @@
 spar_benchmark_russell_p_r1000 = "RUSSELL_P:R.2000"
 spar_benchmark_russell_prefix = "RUSSELL"
 spar_benchmark_russell_return_type = "GTR"
-vault_default_document = "PA3_DOCUMENTS:DEFAULT"
-vault_default_account = "Client:/analytics/data/US_MID_CAP_CORE.ACTM"
-vault_start_date = "FIRST_REPOSITORY"
-vault_end_date = "LAST_REPOSITORY"
+vault_default_document = "Client:/aapi/VAULT_QA_PI_DEFAULT_LOCKED"
```

Comment:
I can't comment on the exact line. But can you please change the environment variables to `ANALYTICS_API_QA_USERNAME_SERIAL` and `ANALYTICS_API_QA_PASSWORD` or something similar?

Reply:
It's in `common_parameters` 😉